metadata-ingestion/src/datahub/ingestion/source/elastic_search.py

```diff
@@ -299,9 +299,10 @@ class ElasticsearchSourceConfig(PlatformInstanceConfigMixin, EnvConfigMixin):
 
     profiling: ElasticProfiling = Field(
         default_factory=ElasticProfiling,
+        description="Configs to ingest data profiles from ElasticSearch.",
     )
     collapse_urns: CollapseUrns = Field(
-        default_factory=CollapseUrns,
+        default_factory=CollapseUrns, description="Urn of collapse."
```

Comment:
In this case the problem seems to be in doc generation instead. CollapseUrns. urns_suffix_regex  is documented properly. This documentation does not really explain the functionality.

Reply:
Same for this https://github.com/datahub-project/datahub/pull/9690/files ?

Reply:
That one I merged in. But `urn of collapse` does not really tell what this is doing. The underlying model has more details which are useful but are not showing up in the docs. If you feel comfortable reviewing the docgen code and making changes in that please do. Otherwise someone in our team can attempt that when they get a chance.